docs/how-to-guides/use-connector-management.md

```diff
@@ -0,0 +1,246 @@
+
+  'genkp.people.with' = '#{Internet.uuid}',
+  'genv.people.name.with' = '#{Name.full_name}',
+  'genv.people.creditCardNumber.with' = '#{Finance.credit_card}',
+
+  'global.throttle.ms' = '500'
+);
+```
+
+## Modes
+
+Before you can use connector management, you need to decide what mode you want to run connectors in. ksqlDB can run connectors in two different modes: **embedded** or **external**. This controls how and where the connectors are executed. The way in which you configure ksqlDB's server determines which mode it will use. All nodes in a single ksqlDB cluster must use the same mode.
+
+Regardless of which mode you use, the syntax to create and use connectors is the same.
+
+## Embedded mode
+
+In embedded mode, ksqlDB runs connectors directly on its servers. This is convenient because it reduces the number of moving parts that you need to manage in your infrastructure. Embedded mode is highly useful for development, testing, and production workloads that have light/moderate data volumes. Use this mode when you don't need to scale your ingest/egress capacity independently from your processing capacity. When you use embedded mode, ksqlDB server is actually running a {{ site.kconnectlong }} server in distributed mode.
+
+### Dowloading connectors
+
+Before you can use a connector, you need to download it prior to starting ksqlDB. A downloaded connector package is essentially a set of jars that contain the code for interacting with the target data store.
+
+The easiest way to download a connector is to use [`confluent-hub`](https://docs.confluent.io/current/connect/managing/confluent-hub/client.html), a utility program distributed by Confluent.
+
+Create a directory for your connectors:
+
+```
+mkdir confluent-hub-components
+```
+
+Run the following command to get the [Voluble](https://github.com/MichaelDrogalis/voluble) data generator connector:
+
+```
+confluent-hub install --component-dir confluent-hub-components --no-prompt mdrogalis/voluble:0.3.0
+```
+
+After running this command, `confluent-hub-components` should contain the Voluble jars. If you are running in clustered mode, you must install the connector on every server.
+
+When you have all the connectors that you need, configure ksqlDB to find them.
+
+!!! important
+    You must restart all of the ksqlDB servers to finish installing the new connectors.
+
+### Configuring ksqlDB
+
+You control whether ksqlDB uses embedded mode by supplying server configuration properties to ksqlDB. If any {{ site.kconnect }}-related properties are present, which are properties prefixed with `ksql.connect.*`, ksqlDB uses these and applies them to the embedded {{ site.kconnect }} server. Although embedded mode eases the operational burden of running a full {{ site.kconnectlong }} cluster, it doesn't dilute {{ site.kconnect }}'s power. Any property that you can configure for a regular {{ site.kconnectlong }} cluster can also be configured for embedded mode.
```

Comment:
This isn't exactly true - the docker image has a special configuration script that will look for any `ksql.connect.*` properties. If you're not using docker, you explicitly need to specify the `ksql.connect.worker.config`

Reply:
Thanks for clarifying! I reworded. What do you think?